src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -961,15 +961,27 @@ public static bool IsDefaultExpressionContext(this SyntaxTree syntaxTree, int po
             {
                 case SyntaxKind.LessThanToken:
                 case SyntaxKind.CommaToken:
-                    return token.Parent is FunctionPointerTypeSyntax;
+                    return token.Parent is FunctionPointerParameterList;
             }
 
-            return token.Parent is ParameterSyntax { Parent: FunctionPointerTypeSyntax _ };
+            return token.IsFunctionPointerParameterOrReturnType();
 #else
             return false;
 #endif
         }
 
+        public static bool IsFunctionPointerParameterOrReturnType(this SyntaxToken syntaxToken)
```

Comment:
Was this being used elsewhere or can it be inlined? (Sorry for the dumb question but it seems text search in my browser isn't being consistent so...)

Reply:
I thought it was being used elsewhere, but I apparently removed that need.

---
In reply to: [451068913](https://github.com/dotnet/roslyn/pull/45684#discussion_r451068913) [](ancestors = 451068913)